vela/admin_test.go

```diff
@@ -237,10 +258,11 @@ func TestAdmin_Secret_Update_200(t *testing.T) {
 	_ = json.Unmarshal(data, &want)
 
 	req := library.Secret{
-		Name:   String("foo"),
-		Value:  String("bar"),
-		Events: &[]string{"barf", "foob"},
```

Comment:
Nice

Reply:
just realized i removed the "barf" here. call me the janitor from now on